default-plugins/tab-bar/src/line.rs

```diff
@@ -144,8 +144,10 @@ fn add_next_tabs_msg(
     title_bar.push(right_more_message);
 }
 
-fn tab_line_prefix(palette: Palette) -> LinePart {
-    let prefix_text = " Zellij ".to_string();
+fn tab_line_prefix(session_name: Option<&str>, palette: Palette) -> LinePart {
```

Comment:
Please do change this part back the way you had it in the prior commit, 
we now have a more clear idea of how to potentially
configure plugins, but it probably still will take some more time.

Until then we would like to show as much information as possible.

Reply:
I've gone ahead and reverted that commit, so it's back to the old behavior